src/SLCore.UnitTests/SLCoreInstanceHandleTests.cs

```diff
@@ -405,6 +410,7 @@ public void Dispose_NotInitialized_DoesNothing()
         });
         jarLocator.ListJarFiles().Returns(JarList);
         activeSolutionBoundTracker.CurrentConfiguration.Returns(new BindingConfiguration(Binding, SonarLintMode.Connected, "dir"));
+        slCoreRuleSettings?.RulesSettings.Returns(new Dictionary<string, StandaloneRuleConfigDto>());
```

Comment:
No unit tests where you actually verify that this value is ever used anywhere

Reply:
Good point.

Reply:
Done.